recipe/meta.yaml

```
@@ -151,28 +151,29 @@ outputs:
         - lib/R/lib
     requirements:
       build:
-        - {{ compiler('m2w64_c') }}          # [win]
-        - {{ compiler('m2w64_cxx') }}        # [win]
+        - libcxx                       # [aarch64 or ppc64le]      
```

Why do you need `libcxx` here in the first place?

---

Was following what was done in [the recommended example](https://github.com/conda-forge/pp-sketchlib-feedstock/pull/40/commits/c17c59dde8a7501697a62b191e444c7267fcdf70). It seemed to imply that the `libcxx` was necessary to enforce the correct version, but maybe I misinterpreted?

@jakirkham am I doing this correctly? Doesn't actually appear to have changed the compiler version, leading to `gxx*=10` with `libcxx=12`.

---

Yeah it's a workaround for a bug encountered with the newer Clang version

That said, think we want this on macOS

```suggestion
        - libcxx                       # [osx]      
```

---

Scratch that sounds like the issue was with GCC (misunderstood that), in which case this shouldn't be here after all

```suggestion
```